lsp-mode.el

```diff
@@ -8102,7 +8102,7 @@ This avoids overloading the server with many files when starting Emacs."
                                     (let ((res (with-current-buffer buf
                                                  ,form)))
                                       (cond
-                                       ((eq res :optional) (propertize "NOT AVAILABLE (OPTIONAL)" 'face 'warning))
+                                       ((eq res :optional) (propertize "OPTIONAL" 'face 'warning))
```

Comment:
Perhaps we should add `:not-available` option here? The `gccemacs` is not available in all Emacs release no

Reply:
the gccemacs was merged into master, does it affect this check somehow?

Reply:
No, native compilation is still optional feature of Emacs